pkg/controllers/provisioning/v1alpha1/reallocation/controller.go

```diff
@@ -66,47 +65,20 @@ func (c *Controller) Reconcile(object controllers.Object) error {
 	provisioner := object.(*v1alpha1.Provisioner)
 	ctx := context.TODO()
 
-	// 1. Get underutilized nodes and label them underutilized
-	underutilized, err := c.filter.GetUnderutilizedNodes(ctx, provisioner)
-	if err != nil {
-		return fmt.Errorf("listing underutilized nodes, %w", err)
+	// Reconcile
+	// 1. Get Underutilized Nodes
+	// 2. Mark any nodes underutilized, and set TTL
+	// 3. Reevaluate any existing underutilized nodes
+	if err := c.utilization.Reconcile(ctx, provisioner); err != nil {
+		return fmt.Errorf("reconciling utilization sub-controller, %w", err)
 	}
 
-	// 2. Set TTL and label underutilized nodes
-	if err := c.annotator.MarkUnderutilized(ctx, c.filter.GetTTLableNodes(underutilized), *provisioner.Spec.TTLSeconds); err != nil {
-		return fmt.Errorf("adding ttl and underutilized label, %w", err)
+	// Reconcile
+	// 1. Get Expired Nodes
```

Comment:
I might leave these details inside the terminator, rather than stating the steps outside.

Reply:
If these details ever change, we'll need to remember to update here (likely will get stale).

Reply:
Since I have the details in the controllers, I'll just remove them. Thanks.